docs/book/src/tasks/experimental-features/runtime-sdk/implement-topology-mutation-hook.md

```diff
@@ -86,7 +86,7 @@ status:
     variables:
```

Comment:
@vincepri would really appreciate if you have the time to review the API, particularly how the variables appear in ClusterClass `.status` and how they can be defined by users in Cluster `.spec.topology`.

We had a discussion about it  - with @sbueringer, @fabriziopandini, @ykakarap -  but it would be good to get another set of eyes :slightly_smiling_face:

Reply:
+1 to the current naming from my side as discussed